osx/7zip/License.txt

```diff
@@ -1,17 +1,21 @@
-  7-Zip source code
-  ~~~~~~~~~~~~~~~~~
+  7-Zip
+  ~~~~~
   License for use and distribution
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-  7-Zip Copyright (C) 1999-2016 Igor Pavlov.
+  7-Zip Copyright (C) 1999-2021 Igor Pavlov.
 
-  Licenses for files are:
+  The licenses for 7zz file are:
```

Comment:
typo? (7zz)

Reply:
No, for some reason the osx binary is called `7zz` now... No clue why. The license file is just copied from the official release.